packages/sdk/client/src/packlets/client/client.ts

```diff
@@ -146,6 +146,7 @@ export class Client {
 
     await this._halo.open();
     await this._echo.open();
+    this._mesh.open();
```

Comment:
always async open/close (if not now, i'm sure it will become so set the pattern)